bundles/aero.minova.rcp.rcp/src/aero/minova/rcp/rcp/handlers/TraverseListenerImpl.java

```diff
@@ -0,0 +1,259 @@
+package aero.minova.rcp.rcp.handlers;
+
+import java.util.List;
+import java.util.Locale;
+
+import org.eclipse.core.runtime.preferences.InstanceScope;
+import org.eclipse.e4.core.services.log.Logger;
+import org.eclipse.e4.ui.model.application.ui.MUIElement;
+import org.eclipse.e4.ui.model.application.ui.basic.MWindow;
+import org.eclipse.e4.ui.model.application.ui.menu.MToolBar;
+import org.eclipse.e4.ui.model.application.ui.menu.MToolBarElement;
+import org.eclipse.e4.ui.model.application.ui.menu.impl.HandledToolItemImpl;
+import org.eclipse.e4.ui.workbench.modeling.EModelService;
+import org.eclipse.e4.ui.workbench.modeling.EPartService;
+import org.eclipse.nebula.widgets.opal.textassist.TextAssist;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.TraverseEvent;
+import org.eclipse.swt.events.TraverseListener;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Widget;
+import org.osgi.service.prefs.Preferences;
+
+import aero.minova.rcp.model.form.MDetail;
+import aero.minova.rcp.model.form.MField;
+import aero.minova.rcp.model.form.MPage;
+import aero.minova.rcp.preferences.ApplicationPreferences;
+import aero.minova.rcp.preferencewindow.builder.DisplayType;
+import aero.minova.rcp.preferencewindow.builder.InstancePreferenceAccessor;
+import aero.minova.rcp.rcp.accessor.AbstractValueAccessor;
+import aero.minova.rcp.rcp.widgets.Lookup;
+
+public class TraverseListenerImpl implements TraverseListener {
+
```

Comment:
Man könnte noch einbauen, dass man sich immer das letzte Feld merkt. Sollte man ein Feld verlassen (mittels Tab oder Enter) könnten wir uns dann dieses Feld als PreviousField merken. Dann brauchen wir nicht immer durch alle Felder der Maske springen. Wenn es kein PreviousField gibt oder es ist null, springen wir in den Code zu Ermittlung des Feldes.

Reply:
Wird in einem eigenen Issue behandelt.

Reply:
#434